platformio.ini

```diff
@@ -591,6 +591,7 @@ build_flags     = -DTTGO=1
                   ${dev_esp32.build_flags}
 lib_deps        = ${dev_esp32.lib_deps}
                   ${ttgo_deps.lib_deps}
+board_build.partitions = config/partitions_custom_noota.csv
```

Comment:
Turning my last comment into a suggestion:

```suggestion
```

Again, if TTGO builds without it.